ansible/envs/staging/group_vars/all.yml

```diff
@@ -1,5 +1,10 @@
 ---
 
+# Fetch all the group params from AWS SSM
+ssm_all: "{{ lookup('aws_ssm', '/staging/ansible/all/', region='us-west-1', shortnames=true, bypath=true, recursive=true) }}"
```

Comment:
If this becomes an issue, because people without access to SSM want to deploy staging services, we can just move this into the group vars for the dev-desktops. I want to deploy the agent on staging _somewhere_, but it doesn't have to be on all machines necessarily.